src/python/library/tritonclient/grpc/_infer_input.py

```diff
@@ -30,7 +30,7 @@
 from tritonclient.grpc import service_pb2
 from tritonclient.utils import *
 
-from ._utils import get_data_type_byte_size, num_elements, raise_error
+from ._utils import num_elements, raise_error
```

Comment:
I see changes still being actively made - what is the current status of this PR? Is it passing all tests?

Reply:
Just updated pipeline ID. Some tests failed but not related to this change.